src/containers/auth/login/index.tsx

```diff
@@ -5,7 +5,7 @@ import { useTranslation } from 'react-i18next';
 
 import { Routes } from '@utilities';
 import { history } from '@store/index';
-import { AuthActionType } from '@store/auth';
+import { AuthActionType } from '@store/enums';
```

Comment:
Again, should be exported from the `auth` branch :)

Reply:
nope